code/asteroid/asteroid.h

```diff
@@ -54,13 +53,20 @@ typedef struct asteroid_split_info {
 	int		max;				//maximum asteroids to spawn
 } asteroid_split_info;
 
+// Data structure for storing asteroid subtype info. POFs, model pointer, model num, etc.
+typedef struct asteroid_subtype_info {
+	char        pof_filename[MAX_FILENAME_LEN];
+	polymodel*  model_pointer;
```

Comment:
I'm a bit wary on saving model pointers, especially when we also store the index which should be used to retrieve them when accessing the model, but since the old code did so too, I guess it's fine for now...

Reply:
I'm with you. I don't have a design in my head to undo that and I can't fully invest time into refactoring that right now either. Good thing to do in a follow-up, though.